src/ui/public/notify/notifier.js

```diff
@@ -230,7 +230,7 @@ Notifier.config = {
   errorLifetime: 300000,
   warningLifetime: 10000,
   infoLifetime: 5000,
-  setInterval: window.setInterval,
+  setInterval: window.setInterval.bind(window),
```

Comment:
`clearInterval` probably does too. The code path I was testing with the Getting Started page ended up exercising `setInterval`, so I fixed that one. I will fix `clearInterval` as well. Thanks for pointing it out.